scripts/cypress/src/base.config.ts

```diff
@@ -56,7 +56,9 @@ const cypressWebpackConfig = (): Configuration => {
 
 type BaseConfig = Omit<Cypress.ConfigOptions, 'component'> & {
   component: Omit<Cypress.ConfigOptions['component'], 'devServer'> & {
-    specPattern?: string[];
+    specPattern?: Cypress.ResolvedConfigOptions['specPattern'];
```

Comment:
this is starting to get unnecesariliy verbose,

we can change the whole signature to following:

```ts
interface BaseConfig extends Cypress.ConfigOptions {
  component: Cypress.Config['component'] & {
    devServer: {
      bundler: 'webpack';
      framework: 'react';
      webpackConfig: Configuration;
    };
  };
}
```

Reply:
This is actually great call! Fixed!